### PR TITLE
Support Aqara E1 2 gang switch (without neutral) QBKG39LM

### PR DESCRIPTION
**Pull request by kirovilya**
https://github.com/Koenkk/zigbee2mqtt/issues/7625

### CODE REVIEW

**Comment by kirovilya** [comment]
there are problems with exposes for relay decoupling.
but if set with mqtt topics, then everything works

File: devices/xiaomi.js [hunk 1]
```diff
@@ -1086,4 +1086,35 @@ module.exports = [
...
+        description: 'Aqara E1 2 gang switch (without neutral)',
+        fromZigbee: [fz.on_off, fz.xiaomi_multistate_action],
+        toZigbee: [tz.on_off, tz.xiaomi_switch_operation_mode, tz.xiaomi_switch_power_outage_memory],
+        meta: {multiEndpoint: true},
+        endpoint: (device) => {
+            return {'left': 1, 'right': 2};
+        },
+        exposes: [
+            e.switch().withEndpoint('left'), e.switch().withEndpoint('right'),
+            exposes.binary('operation_mode', ea.ALL, {state: 'control_relay'},
```
Comment (nurikk) [inline devices/xiaomi.js:1102]: Note: Do not merge this, fix is coming

**Comment by kirovilya** [comment]
Mistake. Everything is working. Can merge

File: devices/xiaomi.js (hunk 1 above)
Comment (kirovilya) [reply devices/xiaomi.js:1102]: no fix - all works :)

**Comment by nurikk** [comment]
Plain json objects never meant to be used as enum values, only generic as strings/numbers/booleans

27536944-DC3C-4B1B-9D21-CBAAB2982F70
@kirovilya @Koenkk

**Comment by Koenkk** [comment]
@nurikk thanks, reverted it, @kirovilya can you create a new PR and use composite instead?

**Comment by kirovilya** [comment]
the composite field looks even weirder than mine variant :)

**Comment by kirovilya** [comment]
ok 
https://github.com/Koenkk/zigbee-herdsman-converters/pull/2715

I think the error with repeating endpoints looks much worse :)
image

**Pull request merged**